service-tests/github.js

```diff
@@ -9,13 +9,23 @@ const {
   isFormattedDate,
   isVPlusDottedVersionAtLeastOne
 } = require('./helpers/validators');
+const colorscheme = require('../lib/colorscheme.json');
 
 const t = new ServiceTester({ id: 'github', title: 'Github' });
 module.exports = t;
+const colorsB = Object.assign({}, ...Object.keys(colorscheme).map(color => ({ [color]: colorscheme[color].colorB })));
 
 t.create('License')
-  .get('/license/badges/shields.json')
-  .expectJSONTypes(Joi.object().keys({ name: 'license', value: Joi.string() }));
+  .get('/license/badges/shields.json?style=extended')
```

Comment:
I'm not fully certain introducing new style the best way to test colors. What do you think about this?

Reply:
It's not a bad approach, as long as we only use it internally. How about we call it `_shields_test` instead of `extended`, to signal to others that they shouldn't use it?

If folks externally want to use it, we really should just add those properties to the json badge.

Reply:
Since nobody currently wants this feature externally let's start with naming it `_shields_test`.